### DISCUSSION TITLE
Integration with scikit-garden and onelearn

### DISCUSSION THREAD

**Discussion by elinama** [online-ml/river#1139] [Ideas]
Hi,
Are you planning to integrate with scikit-garden? In addition, I've seen you were going to incorporate with https://github.com/onelearn/onelearn/tree/master/onelearn and already have permission from the author, but I didn't see it has been done.

Thanks!

**Comment by MaxHalford** [comment]
Hey there @elinama! @AlexandreChaussard is currently working on PR #1131, which will be reviewed by @smastelini. Exciting stuff :)

**Comment by elinama** [comment]
Sounds great, thanks!
